modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java

```diff
@@ -46,8 +51,45 @@ public MongoDBContainer(final DockerImageName dockerImageName) {
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
 
         withExposedPorts(MONGODB_INTERNAL_PORT);
-        withCommand("--replSet", "docker-rs");
-        waitingFor(Wait.forLogMessage("(?i).*waiting for connections.*", 1));
+        configure();
```

Comment:
```suggestion
```

Reply:
configure is part of the Testcontainers lifecycle. So, there is no need to call it directly.